pkg/osquery/table/platform_tables_darwin.go

```diff
@@ -4,27 +4,27 @@ package table
 
 import (
 	"github.com/go-kit/kit/log"
-	"github.com/kolide/osquery-go"
+	osquery "github.com/kolide/osquery-go"
```

Comment:
no need to alias, the import was already there and worked.

Reply:
^ weird, i did not do this manually. i'll remove it